ajax/libs/firebase/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "firebase",
+  "filename": "firebase.js",
+  "description": "Firebase JavaScript library for web and Node.js",
+  "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
+  "homepage": "https://firebase.google.com/",
+  "keywords": [
+    "authentication",
+    "database",
+    "Firebase",
+    "firebase",
+    "realtime",
+    "storage"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/firebase/firebase-js-sdk.git"
+  },
+  "npmName": "firebase",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "firebase*.+(js|map)"
+      ]
+    },
+    {
+      "basePath": "lib",
+      "files": [
+        "firebase*.+(js|map)"
+      ]
+    },
+    {
+      "basePath": "dist",
+      "files": [
+        "firebase*.+(js|map)"
+      ]
+    }
```

Comment:
These seem to match nothing, are they for previous versions?

Reply:
Yes, for version below 3.0.0